tests/integration/targets/ansible_galaxy_install/aliases

```diff
@@ -0,0 +1,2 @@
+destructive
+shippable/posix/group4
```

Comment:
```suggestion
shippable/posix/group3
```
group 4 does not exist.

Reply:
At least this won't happen again, since we now have a sanity check which will scream if this happens :)